### ISSUE TITLE
Task missing Candidate properties

### ISSUE THREAD

**Issue by davidcanonieto** [Activiti/Activiti#2405]
In order to know if a task is claimable by a user the task details must provide these flags that were present before on APS 1.

    managerOfCandidateGroup: boolean;
    memberOfCandidateGroup: boolean;
    memberOfCandidateUsers: boolean;

managerOfCandidateGroup  = current user is the manager of a Candidate Group
memberOfCandidateGroup  = current user is part of a Candidate Group
memberOfCandidateUsers  = current user is part of Candidate Users

If any of these properties is true then the task can be claimed by the user.

**Comment by salaboy** [comment]
@davidcanonieto can you elaborate the use case for this one? 
Currently we are not providing this data due as a user you can see or not see a task. That means that a false in any of those properties will make the task invisible for you, hence you will not be able to claim it.

**Comment by davidcanonieto** [comment]
This (standalone) task has no assignee. Therefore it should not be visible but it is.  Call: .../simpleapp/query/v1/tasks/f98bd154-7628-11e9-86ab-0a5864600524

**Comment by salaboy** [comment]
@davidcanonieto that is not clarifying much.. 
It looks like the owner is being used for that -> owner: "admin.adf" 
Are you logged in with "admin.adf" ?

**Comment by davidcanonieto** [comment]
That is correct @salaboy I'm logged in with admin.adf. I believe that task should not be visible but i might be mistaken, should it?

**Comment by salaboy** [comment]
It is not visible from anyone else besides the one that creates it and ADMINs. If you log in with another user can you see it?

**Comment by salaboy** [comment]
@davidcanonieto also.. how is this now related with the original title and description of the issue?
